bufferGenerators.js

```diff
@@ -1,75 +1,85 @@
-  }
+function uuidBuffer(uuid) {
+  return Buffer.from(uuidParse.parse(uuid))
+}
 
-  static positionBuffer(position) {
-    var x = position.x
-    var y = position.y ? position.y : 16
-    var z = position.z
-    var firstInt32 = ((x & 0x03FFFFFF) << 6) | ((y & 0x0FC0) >>> 6)
-    var secondInt32 = ((y & 0x3F) << 26) | (z & 0x03FFFFFF)
+function positionBuffer(position) {
+  var x = position.x
+  var y = position.y ? position.y : 16
+  var z = position.z
+  var firstInt32 = ((x & 0x03FFFFFF) << 6) | ((y & 0x0FC0) >>> 6)
+  var secondInt32 = ((y & 0x3F) << 26) | (z & 0x03FFFFFF)
 
-    var buf = Buffer.alloc(8)
-    buf.writeInt32BE(firstInt32)
-    buf.slice(4).writeInt32BE(secondInt32)
-    return buf
-  }
+  var buf = Buffer.alloc(8)
+  buf.writeInt32BE(firstInt32)
+  buf.slice(4).writeInt32BE(secondInt32)
+  return buf
+}
 
-  static varIntBuffer(value) {
-    var byteSize = BufferGenerators.varIntSizeOf(value)
-    var bs = new ByteStream(Buffer.alloc(byteSize));
-    bs.writeVarInt(value);
-    return bs.buffer
-  }
+function varIntBuffer(value) {
+  var byteSize = varIntSizeOf(value)
+  var bs = new ByteStream(Buffer.alloc(byteSize));
+  bs.writeVarInt(value);
+  return bs.buffer
+}
 
-  static arrayBuffer(values, f){
-    return Buffer.concat(values.map(f))
-  }
+function arrayBuffer(values, f){
+  return Buffer.concat(values.map(f))
+}
 
-  static floatBuffer(value) {
-    var b = Buffer.alloc(4)
-    b.writeFloatBE(value, 0)
-    return b
-  }
+function floatBuffer(value) {
+  var b = Buffer.alloc(4)
+  b.writeFloatBE(value, 0)
+  return b
+}
 
-  static doubleBuffer(value) {
-    var b = Buffer.alloc(8)
-    b.writeDoubleBE(value, 0)
-    return b
-  }
+function doubleBuffer(value) {
+  var b = Buffer.alloc(8)
+  b.writeDoubleBE(value, 0)
+  return b
+}
 
-  static intBuffer(value) {
-    var b = Buffer.alloc(4)
-    b.writeInt32BE(value)
-    return b
-  }
+function intBuffer(value) {
+  var b = Buffer.alloc(4)
+  b.writeInt32BE(value)
+  return b
+}
 
-  static shortBuffer(value) {
-    var b = Buffer.alloc(2)
-    b.writeInt16BE(value)
-    return b
-  }
+function shortBuffer(value) {
+  var b = Buffer.alloc(2)
+  b.writeInt16BE(value)
+  return b
+}
 
-  static unsignedByteBuffer(value) {
-    var b = Buffer.alloc(1)
-    b.writeUInt8(value)
-    return b
-  }
+function unsignedByteBuffer(value) {
+  var b = Buffer.alloc(1)
+  b.writeUInt8(value)
+  return b
+}
 
-  static varIntSizeOf(x) {
-      return x < 2 ? 1 : Math.ceil(Math.log2(x)/7);
-  }
+function varIntSizeOf(x) {
+  return x < 2 ? 1 : Math.ceil(Math.log2(x)/7);
 }
 
+module.exports = {
+  lengthPrefixedStringBuffer: lengthPrefixedStringBuffer,
+  uuidBuffer: uuidBuffer,
+  positionBuffer: positionBuffer,
+  varIntBuffer: varIntBuffer,
+  arrayBuffer: arrayBuffer,
+  floatBuffer: floatBuffer,
+  doubleBuffer: doubleBuffer,
+  intBuffer: intBuffer,
+  shortBuffer: shortBuffer,
+  unsignedByteBuffer: unsignedByteBuffer,
+  varIntSizeOf: varIntSizeOf
+}
```

Comment:
maybe a slightly pedantic change, just seems like this class with all static methods isn't very javascripty and is also a snowflake that we use nowhere else, so now I'm just passing back an object with all the functions